docs/site/src/contributor/how/continuous-integration.md

```diff
@@ -0,0 +1,3 @@
+# Continuous Integration
+
+TODO: Information about our Continuous Integration pipeline.
```

Comment:
Probably not necessary to capitalise here (since "continuous integration" is not a proper noun):

```suggestion
TODO: Information about our continuous integration pipeline.
```